### ISSUE TITLE
Request: `:top_center` and `:bottom_center`

### ISSUE THREAD

**Issue by sodapopcan** [srcrip/live_toast#2]
My current toasts show up in the bottom centre.  Obviously this isn't super desirable if you have multiple toasts, though my project doesn't need them (I'm more after the auto-hiding).

**Comment by srcrip** [comment]
Will do! Sonner has this option and I would like to have it as well.

**Comment by sergchernata** [comment]
@srcrip do you need a PR for this or are you already working on it?

**Comment by srcrip** [comment]
Hi @sergchernata,

I can knock it out quickly, but I'm working on a few higher priority issues that are more complex first so if you would like to make a PR it will be appreciated too!

**Comment by sergchernata** [comment]
I ended up just wrapping the `LiveToast` component into a `div` and positioning that in the center.

```
<div class="absolute bottom-0 left-1/2 -translate-x-1/2">
  <LiveToast.toast_group flash={@flash} connected={assigns[:socket] != nil} toast_class_fn={&Front.Helpers.toast_class_fn/1} />
</div>
```

**Comment by srcrip** [comment]
implemented in #29 (thank you @nallwhy).
